tests/test_data.py

```diff
@@ -1473,6 +1475,55 @@ def test_gather_time_function(self):
             assert ans == np.array(None)
 
 
+class TestAllocators(object):
```

Comment:
move to `test_gpu_common` as `TestCupyAllocator` with an nividia device skip so it's added to GPU CI